tests/zfs-tests/include/libtest.shlib

```diff
@@ -119,6 +119,21 @@ function is_freebsd
 	fi
 }
 
+# Determine if this is a DilOS test system
+#
+# Return 1 if platform DilOS, 0 if otherwise
```

Comment:
~This logic is inverted from all of the other `is_` platform checks.  Can we keep the behavior consistent and return 0 when the platfirm is DilOS.~  Saw you just fixed this, but please update the comment too.

Reply:
fixed